adodbapi/quick_reference.md

```diff
@@ -61,7 +60,7 @@ document.](https://docs.microsoft.com/en-us/sql/connect/oledb/oledb-driver-for-s
 
 - Jet (ACCESS database) and other file datasets (like .xls and .csv) "ACE"
 Provider:
-    - <http://www.microsoft.com/en-us/download/details.aspx?id=13255> a (32
+  - <http://www.microsoft.com/en-us/download/details.aspx?id=13255> a (32
```

Comment:
A better link would be https://learn.microsoft.com/en-us/office/troubleshoot/access/jet-odbc-driver-available-32-bit-version . This link is broken.

Reply:
Done